src/components/subheader/components/descriptionAdress/DescriptionAdress.js

```diff
@@ -0,0 +1,41 @@
+// Libraries
+import PropTypes from 'prop-types'
+
+// reader
+import addressReader from "../../readers/addressReader";
+
+
+function DescriptionAdress(props){
+
+    const { address } = props;
+
+    const name = addressReader.name(address);
```

Comment:
I think the address should only have city, street etc, and name and city should be separate

Reply:
that i think depends on backend data that we are getting, we can get it as a single object or each attributes separately